src/pages/documentation/adobe-campaign-standard/release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## July 12, 2023
+
+### Android Campaign 2.0.5
+
+- Fixed a bug where a bundled asset was not being used in a fullscreen message when the remote image URL was inaccessible.
```

Comment:
I believe bundled image was not being picked up even when specified alone so should we omit `when the remote image URL was inaccessible.`?

```
Fixed an issue where a bundled asset was not being displayed when specified during in-app message authoring on Adobe Campaign Standard UI.
```

Reply:
a bundled image alone is working. the issue fixed occurred only when a remote url was provided with a bundled image.